types/react-virtualized/dist/es/WindowScroller.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Validator, Requireable, PureComponent } from "react";
+import { Validator, Requireable, PureComponent } from 'react';
```

Comment:
Sorry, but you need to change the single quote back to double quotes. Mixed using is not a good idea. You can check the other code files, eg: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-virtualized/dist/es/Table.d.ts#L1

Reply:
The quotes and the formatting below are changed automatically during the pre-commit. Should I skip that ?

Reply:
I guess it is relevant to your local husky/pre-commit configuration, or prettier configuration, or your IDE configuration...

Reply:
It's due to the precise-commits hook, which is used by the entire repo it looks like. It's definitely not something I added.

try cloning the repo, changing something then committing, it will run prettier with the config from the repo root, which uses single quotes.

Maybe we need to just run prettier for all types/react-virtualized ?

It makes no sense to have that commit hook only to bypass it for individual subtrees.

Reply:
Relevant: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35672

Reply:
Aye, I'll approve this first.